Sources/Datadog/Tracing/TracingV2Configuration.swift

```diff
@@ -18,6 +18,21 @@ internal func createTracingConfiguration(intake: URL) -> DatadogFeatureConfigura
     )
 }
 
+internal func mapInternalTags(_ originalTag: String) -> String {
+    switch originalTag {
+    case "application.id":
+        return "_dd.application.id"
+    case "session.id":
+        return "_dd.session.id"
+    case "view.id":
+        return "_dd.view.id"
+    case "user_action.id":
+        return "_dd.action.id"
+    default:
+        return originalTag
+    }
+}
```

Comment:
suggestion/ Similar ask to the one in Logging:
> suggestion/ Could we declare constants for `"application.id"` and `"session.id"` as part of `RUMDependency` namespace defined in `Tracing` module? Similar to how we define [`RUMDependency`](https://github.com/DataDog/dd-sdk-ios/blob/ncreated/RUMM-3320/GH-1292-fix-apm-with-rum/DatadogSessionReplay/Sources/DatadogCoreIntegration/RUMDependency.swift) in Session Replay - single place with listing all attributes that features exchange between each other.

cc @maxep

Reply:
If the constants are shared eg "application_id", declaring them each module (Tracing, Session Replay etc) seems error prone. IMO, it should go atleast in Internal module which is shared by all other modules.

Not related to this, but having an API package for each feature makes these things easier to maintain, where there could be `RUMApi` module with plain structs, APIs and constants (ideal world) which can be taken as dependency by others without all the baggage of RUM.